src/main/java/org/zalando/nakadi/validation/SchemaEvolutionService.java

```diff
@@ -140,18 +143,27 @@ private String formatErrorMessage(final SchemaChange change) {
         return change.getJsonPath() + ": " + errorMessages.get(change.getType());
     }
 
-    private Version.Level semanticOfChange(final List<SchemaChange> changes,
+    private Version.Level semanticOfChange(final EventTypeBase original, final EventTypeBase eventType,
```

Comment:
Also I wonder if it would be better to change the name of the second parameter. The first two parameters are event types, so calling the second one `eventType` is a bit confusing.